Makefile

```diff
@@ -25,6 +25,13 @@ DATAHOST := cert.cloud.redhat.com
 # Used to identify the agency providing the connection broker.
 PROVIDER := Red Hat
 
+# RPM specific variables
+SPECDIR ?= $(PWD)
+SOURCEDIR := $(shell rpmbuild -E %_sourcedir)
+SRCRPMDIR := $(shell rpmbuild -E %_srcrpmdir)
+VERSION := $(shell echo $(VERSION) | awk -F. '{printf("%d.%d.%d", $$1, $$2, $$3+1)}')
+RELEASE = $(shell printf "0.%s.git.%s" $(shell git rev-list $(shell git describe --tags --abbrev=0 --always | tr -d '\n')..HEAD --count | tr -d '\n') $(shell git rev-parse --short HEAD | tr -d '\n'))
```

Comment:
I don't customize the VERSION and RELEASE variables in the yggdrasil-0.2 implementation. When packit sends a build to COPR, it modifies the Release field of the spec file, encoding the commit information.